Form/Type/ChangePasswordFormType.php

```diff
@@ -36,7 +36,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'form.current_password',
             'translation_domain' => 'FOSUserBundle',
             'mapped' => false,
-            'constraints' => new UserPassword(),
+            'constraints' => new UserPassword(array('groups' => $options['validation_groups'])),
```

Comment:
I have updated my PR. If the line seems too complex/long, tell me and I will do it outside of the `new UserPassword()` call.